CHANGELOG.md

```diff
@@ -6,6 +6,42 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 1.0.0
+
+This release upgrades to Stylelint 15 and now includes:
+
+* Checks for media query syntax, rules and values
+* Checks for number precision inside CSS functions
+* Fixes for duplicate rules with intentionally different units
+
+Read about breaking changes in the Stylelint [**Migrating to v15.0.0**](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-15.md#breaking-changes) guide.
+
+### Support for Internet Explorer 8, 9 and 10
+
+Support for IE8–10 has not been a requirement for services since June 2018, so we removed the rule [`selector-pseudo-element-colon-notation`](https://stylelint.io/user-guide/rules/selector-pseudo-element-colon-notation/) preventing double-colon `::before` and `::after` pseudo element notation.
+
+This change was made in [pull request #36: Remove `selector-pseudo-element-colon-notation`](https://github.com/alphagov/stylelint-config-gds/pull/36).
+
+### Checking for invalid property/value pairs
+
+Stylelint 15 adds an optional new rule [`declaration-property-value-no-unknown`](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown) to check for invalid property/value combinations such as:
+
+```css
+.govuk-link--example {
+  top: red;
+}
+```
+
+Do try it on your projects before we enable it in future, but [report any problems on this issue](https://github.com/alphagov/stylelint-config-gds/issues/42) to help us assess its impact.
```

Comment:
Since this isn't actually part of the 1.0.0 changes I'd consider this information that is superfluous for the changelog and we should remove it - it risks detracting from the more significant bits.

I think if we want to try out that rule and get feedback we should do a pre-release instead?

Reply:
Thanks @kevindew yeah it was a [suggestion for feedback](https://github.com/alphagov/stylelint-config-gds/pull/32#discussion_r1266837855)

Happy to remove it in a future pre-release instead